create a new module where you create, edit, delete and consult

EQC16 opened the pull request:
The "videogame" module is a system for managing video game records. It allows users to create new game entries with details like title, genre, release year, and platform. Users can edit existing game information, delete games from the records, and consult (view) details of specific games. The module provides a simple and organized way to store and manage video game data.

EQC16 committed:
create a new module where you create, edit, delete and consult

EQC16 committed:
Merge branch 'master' into feature/add-crud-videogame

And-Y21 commented on controller/videogame.php:
```diff
@@ -0,0 +1,62 @@
+            require 'model/videogameDAO.php';
+            $this->loadModel('videogameDAO');
+            $videogameDAO = new videogameDAO();
+            $videogameDAO->update($data);
```
Cambiar las variables a ingles

EQC16 replied:
Listo se han cambiado las variables a ingles y el id de actualizar tambien se cambio

And-Y21 commented on database/db.sql:
```diff
@@ -132,3 +132,11 @@ CREATE TABLE weather(
+    gender_videogame varchar(50),
+    price_videogame int,
+    release_date_videogame DATE
+);
```
Siempre dejar la línea del último.

EQC16 replied:
Creo que ya quedo listo esta parte

And-Y21 commented on model/videogameDAO.php:
```diff
@@ -0,0 +1,58 @@
+                ':title_videogameActualizar' => $data['title_videogameActualizar'],
+                ':gender_videogameActualizar' => $data['gender_videogameActualizar'],
+                ':price_videogameActualizar' => $data['price_videogameActualizar'],
+                ':release_date_videogameActualizar' => $data['release_date_videogameActualizar']
```
Hay que cambiar a inglés las variables. También el camelCase o snake_case, en id_videogame_update, por ejemplo.

EQC16 replied:
Corrections were made to the code due to problems with variables that were in Spanish to English.

EQC16 committed:
Update db.sql

EQC16 committed:
Update db.sql

Some space errors in the code have been updated.

EQC16 committed:
Update videogameDAO.php


Some errors in the variables were updated and changed from Spanish to English.

EQC16 committed:
Update videogame.php

Corrections were made to the code due to problems with variables that were in Spanish to English.

EQC16 committed:
Update index.php

The Index was updated with the updated variables and some name or variable changes from Spanish to English.

EQC16 committed:
Merge branch 'master' into feature/add-crud-videogame
